File: 380-insert-delete-getrandom-o1/380-insert-delete-getrandom-o1.py
class RandomizedSet(object):

	def __init__(self):
		self.myList = []

	def insert(self, val):
		if val in self.myList:
			return False
		self.myList.append(val)
		return True

	def remove(self, val):
		if val in self.myList:
			self.myList.remove(val)
			return True
		else:
			return False		

	def getRandom(self):
		return random.choice(self.myList)

File: 380-insert-delete-getrandom-o1/380-insert-delete-getrandom-o1.js

var RandomizedSet = function() {
    this.set = [];
};

/** 
 * @param {number} val
 * @return {boolean}
 */
RandomizedSet.prototype.insert = function(val) {
    if (this.set.findIndex(ele => ele ===val) === -1) {
        this.set.push(val);
        return true;
    }
    return false;
};

/** 
 * @param {number} val
 * @return {boolean}
 */
RandomizedSet.prototype.remove = function(val) {
    
    const idx = this.set.findIndex(ele => ele ===val);
    if (idx !== -1) {
        this.set.splice(idx, 1);
        return true;
    }
    return false;
}

/**
 * @return {number}
 */
RandomizedSet.prototype.getRandom = function() {
    function getRandomInt(max) {
      return Math.floor(Math.random() * max);
    }

    return this.set[getRandomInt(this.set.length)];
};

/** 
 * Your RandomizedSet object will be instantiated and called as such:
 * var obj = new RandomizedSet()
 * var param_1 = obj.insert(val)
 * var param_2 = obj.remove(val)
 * var param_3 = obj.getRandom()
 */